src/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import {
+  ActivityBar,
+  SideBarView,
+  ViewControl,
+  ExtensionsViewSection,
+  InputBox,
+} from 'vscode-extension-tester';
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function extensionUIAssetsTest(): void {
+  describe('Verify base assets are available after installation', () => {
+    let view: ViewControl;
+    let sideBar: SideBarView;
+    let quickBox: InputBox;
```

Comment:
It's not clear why this exists. It seems like in every code path it's `undefined`.

Reply:
very good point, unused it is indeed. Removing.